post_build_checks/check_for_casetwins.sh

```diff
@@ -12,6 +12,10 @@ set -euo pipefail
 # We have a list of known problems and we want to be warned if we find
 # that list is expanded.
 
+# Make the script agnostic about whether it is run from top level or
+# its own folder.
+SCRIPT_DIR=$(dirname "$0")
```

Comment:
Breaks on symlinks, a better approach is "$(dirname "$(readlink -f "$0")")" if you can tolerate a dependency on `readlink` else you can do this madness:
`"$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"`
(from https://stackoverflow.com/a/4774063/803976)

Reply:
Thanks!